files/usr/lib/jeos-firstboot

```diff
@@ -220,6 +226,15 @@ if [ -n "$password" ]; then
     echo "root:$password" | run /usr/sbin/chpasswd
 fi
 
+EFIVARS_DIR="/sys/firmware/efi"
+# Look for EFI dir to see if the machine is booted in UEFI mode
+modprobe efivars
+if ! [ "$(ls -A $EFIVARS_DIR)" ]; then
```

Comment:
yeah, I looked again in the YasT code, was going with a comment somewhere saying if the dir is not empty is enough to know, but better to keep in sync with YasT implementation that is well thought